pkg/storage/swift/swift.go

```diff
@@ -343,14 +343,27 @@ func (d *driver) CreateStorage(cr *imageregistryv1.Config) error {
 		return err
 	}
 
+	infra, err := util.GetInfrastructure(d.Listers)
+	if err != nil {
+		return fmt.Errorf("failed to get cluster infrastructure info: %v", err)
+	}
+
+	clusterID := infra.Status.InfrastructureName
```

Comment:
You don't need to assign to an intermediary variable, you can just use infra.Status.InfrastructureName below instead.

Reply:
done :)